### PR TITLE
lesson 1-1

### PR DESCRIPTION
**Pull request by elenamagay**

### CODE REVIEW

**Comment by elenamagay** [comment]
lesson 1-1 was pushed under the main branch by mistake

File: src/App.js [hunk 1]
```diff
@@ -33,7 +33,9 @@ function App() {
       <ul>
         {todoList.map(response => {
           return (
-          <li key={response.id}>{response.title}</li>
+            <li key={response.id}>
```
Comment (raquii) [inline src/App.js:36]: Great work-- really like the use of the arrow function!

**Comment by raquii** [comment]
I am almost positive this file is auto generated this way when you run `npx create-react-app` with the `--template minimal` flag, but as of React 18, `ReactDOM.render()` is no longer supported.

https://github.com/elenamagay/ctd-react-albatross/blob/67d063ea8d9a0cc9bf6f6504ea433e11db200e82/src/index.js#L1-L10

You can change this file to instead use the new `createRoot` function, like so: 

```js
import { createRoot } from 'react-dom/client';

const container = document.getElementById('root');
const root = createRoot(container);

root.render(<App />);
```

📖 If you'd like to read more about this change, you can check out this blog post from React: [Updates to Client Rendering APIs](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis)

**Comment by raquii** [comment]
🎨 **Style**: I would remove the blank lines here in your JSX
https://github.com/elenamagay/ctd-react-albatross/blob/67d063ea8d9a0cc9bf6f6504ea433e11db200e82/src/App.js#L29-L33

**Comment by raquii** [comment]
https://github.com/elenamagay/ctd-react-albatross/blob/67d063ea8d9a0cc9bf6f6504ea433e11db200e82/src/App.js#L3

Use `const` over `let` for objects and arrays!

**Comment by raquii** [comment]
One comment about the PR description-- descriptions are important! 

In a job setting, PRs often come as a result of a "ticket" in a project management tool and developers are working on their own tickets while being asked to review code for tickets that they aren't familiar with the description of. If I put up a blank PR or one that just said "Completed SWA-123", my coworkers would have to go look up my ticket before they could review my code, which they would probably find irritating. 😅

Instead, describe what the objective of your changes were. It could be something as simple as "This PR adds a todo list to the main `<App>` component" or more descriptive if you wish! The more changes in your PR, the more detail you should add. 

💡 **You can always edit your description on GitHub by clicking the triple dots (`...`) in the right corner of your comment, too.**

**Pull request merged**